text-input.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Text Input
+sidebar: home_sidebar
+permalink: /text-input/
+---
+
+* TOC Placeholder
+{:toc}
+
+## Introduction
+
+This page describe how to setup basic text input for Flutter applications.
+
+The
+[`Input`](https://docs.flutter.io/flutter/material/Input-class.html)
+widget under
+[Material](https://docs.flutter.io/flutter/material/material-library.html )
+will probably satisfy most text-input use cases. However, `Input` is coupled
```

Comment:
The `Input` widget implements material design's style. If you want an input widget that does not use material design, see `RawInputLine`.